diff --git a/extensions/lisp-mode/lisp-mode.lisp b/extensions/lisp-mode/lisp-mode.lisp
index 56210c05..cae87651 100644
--- a/extensions/lisp-mode/lisp-mode.lisp
+++ b/extensions/lisp-mode/lisp-mode.lisp
@@ -740,7 +740,7 @@
 (define-command lisp-quickload (system-name)
     ((prompt-for-symbol-name "System: " (buffer-package (current-buffer))))
   (check-connection)
-  (eval-with-transcript `(,(uiop:find-symbol* :quickload :quicklisp) ,(string system-name))))
+  (eval-with-transcript `(,(uiop:find-symbol* :load-system :asdf) ,(string system-name))))
 
 (defun prompt-for-symbol-name (prompt &optional (initial ""))
   (let ((package (current-package)))
diff --git a/extensions/lisp-mode/repl.lisp b/extensions/lisp-mode/repl.lisp
index c62f1241..e3ae96b7 100644
--- a/extensions/lisp-mode/repl.lisp
+++ b/extensions/lisp-mode/repl.lisp
@@ -538,7 +538,7 @@
 
 (define-repl-shortcut quickload ()
   (let ((system (prompt-for-system "Quickload System: ")))
-    (listener-eval (prin1-to-string `(ql:quickload ,system)))))
+    (listener-eval (prin1-to-string `(asdf:load-system ,system)))))
 
 (define-repl-shortcut ls ()
   (insert-character (current-point) #\newline)
diff --git a/src/lem.lisp b/src/lem.lisp
index 4de021bc..5d16d31b 100644
--- a/src/lem.lisp
+++ b/src/lem.lisp
@@ -203,9 +203,7 @@ See scripts/build-ncurses.lisp or scripts/build-sdl2.lisp"
 
 (defun init (args)
   (unless (equal (funcall 'user-homedir-pathname) ;; funcall for sbcl optimization
-                 *original-home*)
-    (when (uiop:featurep :quicklisp)
-      (init-quicklisp (merge-pathnames "quicklisp/" (lem-home)))))
+                 *original-home*))
   (run-hooks *before-init-hook*)
   (unless (command-line-arguments-no-init-file args)
     (load-init-file))
@@ -260,7 +258,7 @@ See scripts/build-ncurses.lisp or scripts/build-sdl2.lisp"
         (t
          (let ((implementation (get-default-implementation :errorp nil)))
            (unless implementation
-             (ql:quickload :lem-ncurses)
+             (asdf:load-system :lem-ncurses)
              (setf implementation (get-default-implementation)))
            (invoke-frontend
             (lambda (&optional initialize finalize)
